cmd/cmdoptions/client.go

```diff
@@ -46,6 +51,10 @@ type ClientOptions struct {
 
 // loadTLSConfig inits a TLS config from the provided cert and key files.
 func (c *ClientOptions) loadTLSConfig() (*tls.Config, error) {
```

Comment:
Before this function would return a non-nil config if `EnableTLS` was false but one of the other TLS options was present like `ClientCertPath`. But now this will return `nil` even when one of those values is set.